tracim/tracim/model/serializers.py

```diff
@@ -138,7 +138,7 @@ def get_converter(cls, context_string, model_class):
         try:
             converter = Context._converters[context_string][model_class]
             return converter
-        except:
+        except KeyError:
```

Comment:
Non related with content view deletion; It seems more correct to catch KeyError instead all errors ?